teuthology/test/task/test_ceph_ansible.py

```diff
@@ -41,6 +41,15 @@ def fake_get_scratch_devices(remote):
         )
         self.patcher_get_scratch_devices.start()
 
+        def fake_teardown(self):
+            return
```

Comment:
I don't quite understand this. My question is about how you're declaring a null function to patch with. By default `patch()` replaces the object being patched with [a MagicMock](http://www.voidspace.org.uk/python/mock/patch.html#mock.patch)